docs/platforms/apple/common/index.mdx

```diff
@@ -1,34 +1,172 @@
-<PlatformContent includePath="getting-started-primer" />
+On this page, we get you up and running with Sentry's Apple SDK, automatically reporting errors and exceptions in your application.
 
-On this page, we get you up and running with Sentry's SDK.
+<Note>
+
+If you don't already have an account and Sentry project established, head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+
+</Note>
+
+<PlatformSection supported={["apple.visionos"]}>
+
+<Alert level="info" title="Experimental Support">
 
-Don't already have an account and Sentry project established? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+The support for [visionOS](https://developer.apple.com/visionos/) is currently experimental.
+Experimental support means it may have bugs. We recognize the irony.
```

Comment:
```suggestion
The support for [visionOS](https://developer.apple.com/visionos/) is currently experimental, which means it may have bugs. We recognize the irony.
```